README.md

```diff
@@ -21,6 +21,13 @@ xhr({
 ```
 
 ## `var req = xhr(options, callback)`
+Using this invocation, `options` should include a `uri` property for the request.
+
+## `var req = xhr(url, callback)`
```

Comment:
I didn't notice that until I opened a preview of the markdown, three headings make a weird first impression. I tested that on a representative group of 2 people ;) and I'll try making it look more natural